CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ option(CCCL_ENABLE_TESTING "Enable CUDA C++ Core Library tests." ON)
 include(CTest)
 enable_testing()
 
+include(cmake/CCCLHideThirdPartyOptions.cmake)
```

Comment:
This needs to be done at the end of the CMakeLists.txt

Variables passed to mark_as_advanced that are not already in the cache are ignored ( policy 102 ).

So we want to do this at the end so these are hidden after first CMake execution.